.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
I think the conflict may be that this file is different from the .gitignore in the master branch, so I would suggest modifying your branch's .gitignore to be the following:
```suggestion
# The following won't be added to git

# IntelliJ specific files
.idea/
target/
codeu-starter-project.iml

# Eclipse specific files
.project/
.settings/
.classpath/
preferences
```
This might resolve the conflict, and note that it includes some more files to ignore on git so that we won't have any Eclipse-specific files like .project or .settings/org.eclipse... in the master branch!